packages/keyring-api/src/eth/rpc/params.ts

```diff
@@ -214,3 +221,11 @@ export const EthSignEip7702AuthorizationParamsStruct = tuple([
 export type EthSignEip7702AuthorizationParams = Infer<
   typeof EthSignEip7702AuthorizationParamsStruct
 >;
+
+/**
+ * Parameters for `eth_getEncryptionPublicKey`.
+ */
+export const EthGetEncryptionPublicKeyParamsStruct = tuple([
+  EthAddressStruct, // address
+  EthGetEncryptionPublicKeyOptionsStruct, // options
```

Comment:
IMO it's not so obvious to know that this parameter is optional, WDYT about moving the `optional(...)` call here instead of having it on the struct?